spec/models/bigbluebutton_recording_spec.rb

```diff
@@ -1039,8 +1039,11 @@
 
     context "if can't find the start time in recordid" do
       let(:recording) { FactoryGirl.create(:bigbluebutton_recording, :recordid => "without-timestamp") }
+      before {
+        @meeting = BigbluebuttonMeeting.create_meeting_record_from_recording(recording)
+      }
       subject { BigbluebuttonRecording.send(:find_matching_meeting, recording) }
-      it { subject.should be_nil }
+      it("creates a meeting for the recording") { subject.should eq(@meeting) }
```

Comment:
When `recording.start_time` is nil, the meeting will not be created.
Reference: https://github.com/mconf/bigbluebutton_rails/pull/186/files#R355
`unless recording.start_time.nil?`